### PR TITLE
Overall polishment and Relation Graph using Cytoscape

### PR DESCRIPTION
**Pull request by TomDijkema**
PR adds overall polishment to the application.
Replaced the old relational graph library with Cytoscape, which should be a bit easier to work with and add images and colors. Graphs now also show annotations as potential linkages.

More polishing will be coming, this PR is to save in between.

### CODE REVIEW

File: src/components/profile/components/user/Passport.tsx [hunk 1]
```diff
@@ -151,7 +151,7 @@ const Passport = () => {
             {!editMode &&
                 <Row>
                     <Col lg={{ span: 8, offset: 2 }}>
-                        <OrganisationLogoImage organisationId={userProfile.organisation?.replace('https://ror.org/', '')} />
+                        <OrganisationLogoImage organisationId={userProfile.organisation?.replace('https://ror.org/', '')} />zz
```
Comment (samleeflang) [inline src/components/profile/components/user/Passport.tsx:154]: There is `zz` here is that intentional?

File: src/components/search/components/IDCard/MapMediaExt.tsx [hunk 2]
```diff
@@ -64,47 +69,53 @@ const MapMediaExt = (props: Props) => {
...
+                                            accessUri={mediaItem.digitalEntity['ac:accessUri']}
+                                            format={mediaItem.digitalEntity['dcterms:format']}
+                                        />
+                                    </div>
                                 );
                             })}
-                        </div>
-                    </Col>
+                        </Col>
+                    {/* </Col> */}
```
Comment (samleeflang) [inline src/components/search/components/IDCard/MapMediaExt.tsx:118]: This Col can be removed?

File: src/components/profile/components/user/Passport.tsx (hunk 1 above)
Comment (TomDijkema) [reply src/components/profile/components/user/Passport.tsx:154]: Good catch! Removed

**Review by samleeflang — APPROVED** [review]
:sauropod:

**Pull request merged**